lib/dao.js

```diff
@@ -1579,8 +1580,8 @@ DataAccessObject._normalize = function(filter, options) {
   }
 
   var handleUndefined = this._getSetting('normalizeUndefinedInQuery');
-  // alter configuration of how removeUndefined handles undefined values
-  filter = removeUndefined(filter, handleUndefined);
+  // alter configuration of how sanitizeQuery handles undefined values
+  filter = this._sanitize(filter, {handleUndefined: handleUndefined});
```

Comment:
can we rename the option from `handleUndefined` to `normalizeUndefinedInQuery`? I find `handleUndefined` slightly confusing (what does it mean to handle undefined values?), plus the setting is already called `normalizeUndefinedInQuery` so let's be consistent.

Reply:
> can we rename the option from `handleUndefined` to `normalizeUndefinedInQuery`? I find `handleUndefined` slightly confusing (what does it mean to handle undefined values?), plus the setting is already called `normalizeUndefinedInQuery` so let's be consistent.

@raymondfeng ☝️ I think you forgot to address this comment.